helm/cluster-autoscaler-app/values.yaml

```diff
@@ -8,17 +8,32 @@ groupID: 65534
 port: 8085
 
 # Make sure to update the docs at https://github.com/giantswarm/docs/blob/master/src/layouts/shortcodes/autoscaler_utilization_threshold.html if you change the defaults below
-configmap:
```

Comment:
We could use already released versions of the `cluster-autoscaler-app` for existing AWS clusters deployed using chartconfigs CRs, and release a new major version of the app with the new values (removing the `configmap` bit from the name of the values).

WDYT?

Reply:
Having these values under configmap is ugly but I'm not sure its worth a breaking change and I want to avoid manual steps during upgrades.

We need to copy these values from the tenant cluster to a new configmap in the control plane cluster.

We could also change the values structure but it adds complexity to the migration process.

Reply:
Ok! Does this mean that I have to revert the `deployment.yaml` changes regarding the extra arguments to keep referencing the values under `configmap`?

Reply:
You know way more about this issue some let's go with your suggestion, but I think it makes sense to have a breaking change when you are migrating from one system to another one. Otherwise, we will keep using the `configmap` key for our values even though this is not needed anymore, inheriting it from the current system.

Reply:
We're just moving from the chartconfig CRD to app CRD. So customers shouldn't have to care about this.

Yes please change the deployment and keep the current values structure for now.

Changing the structure isn't hard for this app but sadly I don't have time to do it. Do we have an ETA yet for when upstream will ship auto discovery?

If we have time and you want to do this. I can help with pointing out what needs to change in the `configmapmigration` resource.

Reply:
Changed